pkg/oci/oci.go

```diff
@@ -478,16 +490,85 @@ func newAuthClient(repository string, authOptions *AuthOptions) (*oras_auth.Clie
 	}, nil
 }
 
+func prepareCheckOpts(ctx context.Context, publicKey string) (*cosign.CheckOpts, error) {
+	pubKey, err := signature.LoadPublicKeyRaw([]byte(publicKey), crypto.SHA256)
```

Comment:
Is it always gonna be `crypto.SHA256` or do we have to make that customizable somehow?